gsad/src/gsad.c

```diff
@@ -542,7 +542,7 @@ init_validator ()
   gvm_validator_add (validator, "chart_init:value", "(?s)^.*$");
   gvm_validator_add (validator, "setting_value", "^.*$");
   gvm_validator_add (validator, "setting_name", "^(.*){0,1000}$");
-  gvm_validator_add (validator, "comment",    "^[-_;':()@[:alnum:]äüöÄÜÖß, \\./]{0,400}$");
+  gvm_validator_add (validator, "comment",    "(?s)^(.*){0,1000}$");
```

Comment:
This makes the use of TextArea for comments obvious, as we don't have line breaks in TextField. But my comment about consistency still holds. I'm not sure what we should do about this.